docs/_source/guides/llms/practical_guides/fine_tune.md

```diff
@@ -434,10 +433,26 @@ record = rg.FeedbackRecord(
 )
 
 rg.add_records([record])
+```
+
+Then push it to Argilla via `push_to_argilla`.
+
+::::{tab-set}
+
+:::{tab-item} Argilla 1.14.0 or higher
+```python
+remote_dataset = dataset.push_to_argilla(name="pre-training")
```

Comment:
Same as with other snippets `remote_dataset` is not being used so maybe it's not necessary to save as a variable here. However, in this specific page the code snippet below seems to use the dataset, so this snippet could be integrated here keeping the `remote_dataset` variable.